src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -34,6 +34,9 @@
     search-source="savedObj.searchSource"
     show-spy-panel="!isFullScreenMode"
     ui-state="uiState"
+    shared-item
```

Comment:
I wonder if we could use consts somehow, so that way if they are ever changed in kibana, it won't break reporting.  But I guess you can't create attributes dynamically, like
```
<visualize {{Consts.SharedItemTag}} />
```
Similar to your worry when I fixed the issue with hiding the reporting link on the dashboard landing page.  It's going to be hard for someone to realize that these attributes are used in reporting and changing them will break it.  Especially if they are here to help a timing issue, knowing when to load the page, tests might not even fail right away.

I wonder if we could use a const if we did something similar to how we use `data-test-subj`.  Though that only works if no one changes the data-test-subj tag itself.

I just could see someone being like, why is this here and what does it do, and it's not very discoverable.